etc/profile-m-z/xarchiver.profile

```diff
@@ -0,0 +1,19 @@
+# Firejail profile for xarchiver
+# Description: Lightweight desktop independent archive manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include xarchiver.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${RUNUSER}
+ignore include disable-shell.inc
+ignore x11
+ignore dbus-user none
+
+noblacklist ${HOME}/.config/xarchiver
+
+dbus-user.talk ca.desrt.dconf.service
```

Comment:
Do we ever enable filtering?

Reply:
> Do we ever enable filtering?

@northboot

You can check what dbus paths are used with d-feet or busctl, then add the
relevant filter/own (/talk) commands:

```
dbus-user filter
dbus-user.own foo.bar.baz
```

See for example:

* <https://github.com/netblue30/firejail/issues/6444#issuecomment-2322650453>

Reply:
> Updated the commands

Still missing a `dbus-user.own` entry.

Reply:
Does it need one?

Reply:
> Does it need one? / have one?

I was thinking that common GUI programs that use dbus would generally register
their own dbus name even if it's not currently used.

But now I see that engrampa.profile and some other profiles have .talk without
.own.

> No, it doesn't have one: `$ dbus-send --print-reply --session
> --dest=org.freedesktop.DBus /org/freedesktop/DBus
> org.freedesktop.DBus.ListNames | grep -i archiver` does not output anything
> (while running the application)

I see, thanks for confirming.